src/assets/js/front-end.js

```diff
@@ -97,6 +99,14 @@ var BoldGrid = BoldGrid || {};
 				};
 			},
 
+			// Set correct height of .demo_store notice.
+			demoStore: function() {
+				if ( 0 !== $( '.demo_store:visible' ).length ) {
```

Comment:
The visible selector is quite resource intensive. Also positioning with Javascript is generally not desirable if it's possible to achieve the effect with CSS. Especially on the front end of a user's site.

I suggest finding an approach that does rely on javascript for css.

Reply:
I was having trouble finding a way to do this that would both 1) keep the notice fixed on the bottom of the window, and 2) ensure that the size of the notice adjusts for the content when resizing the window. 

I will look further into making sure this only runs if they have woocommerce enabled

Reply:
So i was able to get it working using just CSS. It looks like woocommerce's CSS had another selector that was setting the 'top' property. I was able to override that and it displays properly without using JS